### DISCUSSION TITLE
The amount charged for the sponsorship tier is inconsistent with what was expected.

### DISCUSSION THREAD

**Discussion by libukai** [squidfunk/mkdocs-material#5344] [Q&A]
I chose a sponsorship tier of $15 per month, and on the subscription page, it showed that I still needed to pay $4.93 within this annual period. 

<img width="712" alt="CleanShot 2023-04-09 at 23 04 42@2x" src="https://user-images.githubusercontent.com/5654585/230780605-447b994d-35a3-41a3-b57f-175248eaed64.png">

However, when the credit card was charged, it deducted $156.33.

<img width="1138" alt="CleanShot 2023-04-09 at 23 00 22@2x" src="https://user-images.githubusercontent.com/5654585/230780308-8491db64-b47f-492f-8d61-f6f0788d5e2c.png">

 I'm wondering what happened, anyone could help explain it?

**Comment by squidfunk** [comment]
Thanks for (trying?) sponsoring!

The $4,93 is the prorated amount for a month, so your billing cycle would be due in 10 days or so. However, I'm not sure why GitHub then switched to yearly billing. Could be a bug in GitHub's payment processor. I'm sorry for the inconvenience, but we can't help you with anything payment related. I recommend writing to GitHub support, they are helpful and quick to respond.
